meshtastic/tests/test_smoke2.py

```
@@ -2,42 +2,51 @@
+    pytest.fail(f"{' '.join(cmd)} timed out after {timeout_seconds} seconds: {exc}")
+
+
+def _run_meshtastic(
+    cmd: list[str],
+    timeout_seconds: int,
+) -> subprocess.CompletedProcess[str]:
+    """Run a meshtastic CLI command with a timeout and fail cleanly on timeout."""
     try:
-        result = subprocess.run(
-            [meshtastic_bin, "--info"],
+        return subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)